lib/gis/strings.c

```diff
@@ -452,7 +492,40 @@ char *G_strcasestr(const char *str, const char *substr)
     return (char *) q;
 }
 
-int _strncasecmp(const char *x, const char *y, int n)
+/*!
+  \brief Copy string until character found
+
+  The bytes from string src are copied to string dst.  If the character c (as
+  converted to an unsigned char) occurs in the string src, the copy stops and
+  a pointer to the byte after the copy of c in the string dst is returned.
+  Otherwise, n bytes are copied, and a NULL pointer is returned.
+
+  The source and destination strings should not overlap, as the behavior
+  is undefined.
+
+  \param dst destination
+  \param src source
+  \param c stop character
+  \param n max number of bytes to copy
+
+  \return a pointer to the next character in dest after c
+  \return NULL if c was not found in the first n  characters of src
+*/
+static void *G__memccpy(void *dst, const void *src, int c, size_t n)
+{
+    const char *s = src;
+    char *ret;
+
+    for (ret = dst; n; ++ret, ++s, --n) {
+    *ret = *s;
```

Comment:
Sorry! Keep finding new stuff at the last minute. This `for` block needs to be properly indented.

Reply:
I think all this formatting should be handled by the indent rules (https://trac.osgeo.org/grass/wiki/Submitting/C#Indentation) in a separate PR after this one last change.